examples/common/io.ads

```diff
@@ -0,0 +1,22 @@
+
+with Ada.Sequential_IO; Use Ada;
+with Ada.Strings.Unbounded; use  Ada.Strings.Unbounded;
+
+with GL.Types; use GL.Types;
+
+package IO is
```

Comment:
Read_String procedures deleted.
I tried converting to Text_IO but was unsuccessful.
However, in trying to do so I somehow made changes that produce a better rendered result  (possibly due  to Adaerizing some indices).
I may try again after attending to other problems.